src/Controller/CRUDController.php

```diff
@@ -1094,9 +1099,9 @@ protected function getParameter($name)
      * @param int   $status
      * @param array $headers
      *
-     * @return Response with json encoded data
+     * @return JsonResponse with json encoded data
      */
-    protected function renderJson($data, $status = 200, $headers = [])
+    protected function renderJson($data, $status = 200, $headers = []): JsonResponse
```

Comment:
That's a BC break for extending classes, please revert.

Reply:
But please keep the JsonResponse int he phpdoc 👍🏻

Reply:
reverted return-type, updated docblock